main.cpp

```diff
@@ -46,6 +47,29 @@ int main(int argc, char *argv[])
     application.setOrganizationName("Yubico");
     application.setOrganizationDomain("com.yubico");
 
+    QDesktopWidget * desktop = QApplication::desktop();
+
+    // Get x and y coordinates of all monitors
+    QVariantList monitorList;
+    for (qint32 i = 0, screens = desktop->screenCount(); i < screens; i++)  {
```

Comment:
I'm getting a deprecation warning in Qt 5.14:

```
main.cpp:54:55: warning: ‘int QDesktopWidget::screenCount() const’ is deprecated: Use QGuiApplication::screens() [-Wdeprecated-declarations]
   54 |     for (qint32 i = 0, screens = desktop->screenCount(); i < screens; i++)  {
      |                                                       ^
```

Looks like the non-deprecated API is something like this:

```cpp
    for (QScreen* screen : QGuiApplication::screens())  {
        QRect geometry = screen->geometry();
        // ...
        int monitor_x_min = geometry.x();
        // ...
    }
```

Reply:
Good catch!